src/views/v2/events-bar/form.php

```diff
@@ -12,13 +12,18 @@
  * @version 4.9.3
  *
  */
+
+use Tribe\Events\Views\V2\Rest_Endpoint;
 ?>
 <div class="tribe-events-c-events-bar__form">
 	<form
 		class="tribe-common-c-search"
 		method="get"
 		data-js="tribe-events-view-form"
 	>
+		<?php wp_nonce_field( 'wp_rest', 'tribe-events-views[_wpnonce]' ); ?>
```

Comment:
I'd suggest the action be something other than `wp_rest` as the view won't always hit a REST endpoint.

Reply:
Agreed, that is something we will need to address by having this be a method inside of the of another class later, or template.